Source: @pullmoll
Upstream: no
Reason: This test fails for Musl libc and we know we have a working libc

--- a/configure.ac	2020-01-23 00:14:16.000000000 +0100
+++ b/configure.ac	2020-01-30 00:23:00.859196125 +0100
@@ -10233,18 +10233,10 @@
 AC_SUBST(LIBNUMBERTEXT_CFLAGS)
 
 dnl ***************************************
-dnl testing libc version for Linux...
+dnl Always assume we have a working libc
 dnl ***************************************
 if test "$_os" = "Linux"; then
-    AC_MSG_CHECKING([whether libc is >= 2.1.1])
-    exec 6>/dev/null # no output
-    AC_CHECK_LIB(c, gnu_get_libc_version, HAVE_LIBC=yes; export HAVE_LIBC)
-    exec 6>&1 # output on again
-    if test "$HAVE_LIBC"; then
-        AC_MSG_RESULT([yes])
-    else
-        AC_MSG_ERROR([no, upgrade libc])
-    fi
+    HAVE_LIBC=yes; export HAVE_LIBC
 fi
 
 dnl =========================================
